src/main/webapp/go.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Go On Scavenger Hunt</title>
+    <link rel="stylesheet" href="go.css" />
+    <script src="config.js"></script>
+    <script src="go.js"></script>
+  </head>
+  <body>
+    <div class="content">
+      <div class="center-block">
+        <div id="map-area"></div>
+      </div>
+      <div class="center-row">
+        <div id="map-message-area"></div>
+      </div>
+      <form action="/name-data" method="POST">
```

Comment:
I think that there's something non-obvious here about the way this data flows.

When a user submits a guess here, it isn't immediately checked.  Here's what happens:
1) user types a guess and clicks submit
2) /name-data stores that name and redirects back to go.html
3) go.html calls getHunt on load, which makes a call to /go-data to get the scavenger hunt object
4) updateToCurrentState is the method that actually does the checking to see if needs to proceed to the next destination

This might be simpler if this could be handled in one request instead of two, something like /check-name?input&Golden%20gate%20bridge&hunt=5&index=2 that would return whether the guess is correct or not.  In any case, we won't make this blocking, but something to think about!